pep/app/pods/application/route.ts

```diff
@@ -85,16 +87,21 @@ export default class Application extends PageLayout(Route.extend(ApplicationRout
         // since we do not currently cache ajax service requests
         // in the fastboot shoebox (like ember-data does)
 
+        if (this.fastboot.isFastBoot) {
+            this.auth.dontRedirectOnLogin = true;
+        }
+
         if (this.fastboot.isFastBoot && !this.session.isAuthenticated) {
             try {
                 await this.session.authenticate('authenticator:ip');
+                this.fastboot.request.cookies.pep_session = this.cookies.read('pep_session');
+                this.cookies.write('pep_session_fastboot', this.cookies.read('pep_session'), {});
```

Comment:
might be good to add some comments here to explain that this is a workaround/why its needed.